package.json

```diff
@@ -1,33 +1,45 @@
 {
-  "name": "@parity/dapp-addresses",
-  "description": "Parity default addressbook view",
-  "version": "1.99.99",
+  "name": "bond-dapp-addresses",
+  "version": "1.0.0",
+  "description": "address dapp using bonds",
   "main": "index.js",
-  "author": "Parity Team <admin@parity.io>",
-  "maintainers": [],
-  "contributors": [],
-  "license": "GPL-3.0",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/paritytech/dapp-addresses.git"
-  },
-  "keywords": [],
   "scripts": {
     "build": "NODE_ENV=production webpack --colors --progress",
     "watch": "webpack --colors --progress --watch"
   },
+  "author": "Parity Team <admin@parity.io>",
+  "license": "GPL-3.0",
+  "dependencies": {
+    "@parity/api": "^2",
+    "@parity/dapp-accounts": "file:../dapp-accounts",
+    "@parity/ui": "file:../js-ui",
```

Comment:
Point to npm package (^2) - if there are changes, that package should be updated with a PR.